pandas/tests/generic/test_duplicate_labels.py

```diff
@@ -427,7 +427,6 @@ def test_dataframe_insert_raises():
     [
         (operator.methodcaller("set_index", "A", inplace=True), True),
         (operator.methodcaller("set_axis", ["A", "B"], inplace=True), False),
-        (operator.methodcaller("reset_index", inplace=True), True),
```

Comment:
moving this one to a separate test (below) so as to not have too much logic in a test